outline.c

```diff
@@ -135,25 +135,11 @@ _outline_close(OutlineObject* self, PyObject* args)
     return Py_None;
 }
 
-static PyObject*
-_outline_transform(OutlineObject* self, PyObject* args)
-{
-    double a[6];
-    if (!PyArg_ParseTuple(args, "(dddddd)", a+0, a+1, a+2, a+3, a+4, a+5))
-        return NULL;
-
-    ImagingOutlineTransform(self->outline, a);
```

Comment:
I don't think it's ever been used. I've added public APIs for some other functions, but I'm not sure what's expected of this one, so be my guest! :)